### ISSUE TITLE
say i dont have enough ram to play tekkit 2

### ISSUE THREAD

**Issue by LoneWulf2751** [xJon/Tekkit-2#129]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: not applicable

**Help needed**

A clear description of what help is needed.

every time i try to run tekkit 2 it says i do not have enough ram, says im running 1GB when i need 2.5GB, when i have 32GB in my system.
![Screenshot 2022-10-24 170832](https://user-images.githubusercontent.com/116584410/197579234-a954b1a2-f0ac-46a2-a957-e658c03270c4.png)

**Comment by xJon** [comment]
![image](https://user-images.githubusercontent.com/8945347/197581411-9a565131-07df-4650-b181-789b46eb20f9.png)

**Comment by LoneWulf2751** [comment]
i have the required java version and have more than enough RAM, also i cant click the blue link

**Comment by LoneWulf2751** [comment]
![Screenshot 2022-10-24 173252](https://user-images.githubusercontent.com/116584410/197583560-85404061-8391-4c80-ace8-14307900500e.png)

**Comment by xJon** [comment]
Hmm, did you try going to the Java Settings in the Launcher Options to allocate more RAM?

**Comment by Blacksail-1** [comment]
if you go into the launcher setting and change the ram allotment it'll run (i had to do this btw when i got the modpack)

**Comment by Nyun0** [comment]
If you're having both versions (32 and 64 bit) installed, you might want to try the registry tweaks I did here: https://forums.technicpack.net/topic/328467-techniclauncherexe-not-recognizing-java/

But first, open the terminal (cmd.exe) and check what does _java -version_ return. It should be similar to what I have posted in the link I've mentioned. If you do get 64 bit version there it only confirms you would need to make the registry tweaks, as the technic launcher uses the registry entries to determine which Java version you are running.
